generator/src/test/java/com/scottlogic/deg/generator/cucumber/InSet.feature

```diff
@@ -668,18 +669,6 @@ Scenario: Running a 'inSet' request alongside a non-contradicting matchingRegex
        | "test"  |
        | "Testt" |
 
-@ignore
-Scenario: Running a 'inSet' request alongside a contradicting matchingRegex constraint should fail with an error message
```

Comment:
This test should be reinstated and retitled to be a contradicting containingRegex

Reply:
re-instated